include/alpaka/mem/buf/Traits.hpp

```diff
@@ -175,9 +177,10 @@ namespace alpaka
         TPlatform const& platform,
         TExtent const& extent = TExtent())
     {
-        if constexpr(hasMappedBufSupport<TPlatform>)
+        using PlatformForMappedAlloc = Platform<TPlatform>;
```

Comment:
I would suggest
```suggestion
        using Platform = alpaka::Platform<TPlatform>;
```
if it works?

Reply:
See for example https://github.com/alpaka-group/alpaka/blob/ecf917fa2a7d4cfa6e15e827688193c5e77302d5/include/alpaka/test/queue/QueueTestFixture.hpp#L17

Reply:
This should now be consistent with the other usages of the `Platform`-Trait, which all have the `alpaka` namespace prefixed.